src/components/dialog-full-screen/dialog-full-screen.component.js

```diff
@@ -74,21 +77,32 @@ class DialogFullScreen extends Modal {
   /**
    * Overrides the original function to disable the document's scroll.
    */
-  get onOpening() {
+  onOpening() {
+    this.focusTrap = new FocusTrap(this._dialog);
```

Comment:
We always call the following two lines together and the only other method that we call is `removeFocusTrap`.
```js
this.focusTrap = new FocusTrap(this._dialog);
this.focusTrap.setFocusTrap()
```

Could we change it so the interface is

```js
onOpening(){
  this.removeFocusTrap = focusTrap(this._dialog);
}

onClosing(){
  this.removeFocusTrap();
}
```
e.g.

```js
focusTrap(el){
  const Cls = new FocusTrap(cls);
  return Cls.removeFocusTrap.bind(cls);
}
```

What do you think?

Reply:
After I have removed the method `setFocusToFirstElement` there is nonsense to force people to use `focusTrap and then setFocusTrap`. I'll change this